src/server/game/Spells/Auras/SpellAuras.cpp

```diff
@@ -1161,6 +1160,11 @@ bool Aura::CanBeSaved() const
     if (GetCastItemGUID() && IsPermanent())
         return false;
 
+    // don't save liquid auras
+    for (LiquidTypeEntry const* liquid : sLiquidTypeStore)
```

Comment:
thats not a scalable approach

Reply:
So what can be placed instead?

Reply:
maybe a custom spell attribute which is added on startup?

Reply:
Ready, thanks